app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+
+  def create
+    super
+  end
+
+  def sign_up(_name, _resource)
+  end
```

Comment:
Sorry, couldn't understand what this code does here. What's this for?

Reply:
This is overriding existing Sign Up method. It will prevent auto sign in after creating new account. 📌

Reply:
Ah, gotcha.